### ISSUE TITLE
Is project alive

### ISSUE THREAD

**Issue by pedromorgan** [windelbouwman/can-bus-explorer#1]
hi @windelbouwman 

Am looking at this project, but kinda need to make it more of a module. 

Are you accepting patches ?

**Comment by windelbouwman** [comment]
Eh, sure, I could have a look! This project is pretty inactive, but if you find it useful, please give it a shot. I'm curious what you come up with.

**Comment by pedromorgan** [comment]
Main thing is to move stuff into a `can_explorer` directory, so it can be packaged etc for pypi, as i need to import on another curious.cat project

**Comment by windelbouwman** [comment]
You could also copy-paste the sources as needed? Do you think the gui is reusable as a component?

**Comment by pedromorgan** [comment]
>> Do you think the gui is reusable as a component?

Yep, that's kinda why am here i guess. Looked around but nothing in pyqt....

**Comment by pedromorgan** [comment]
Started a PR so u can see what's happening.. https://github.com/windelbouwman/can-bus-explorer/pull/2
